beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -73,66 +73,68 @@ func (vs *Server) getLocalPayloadAndBlobs(ctx context.Context, blk interfaces.Re
 				"Please refer to our documentation for instructions")
 		}
 	default:
-		return nil, nil, false, errors.Wrap(err, "could not get fee recipient in db")
+		return nil, false, errors.Wrap(err, "could not get fee recipient in db")
 	}
 
 	if ok && proposerID == vIdx && payloadId != [8]byte{} { // Payload ID is cache hit. Return the cached payload ID.
 		var pid [8]byte
 		copy(pid[:], payloadId[:])
 		payloadIDCacheHit.Inc()
-		payload, blobsBundle, overrideBuilder, err := vs.ExecutionEngineCaller.GetPayload(ctx, pid, slot)
+		var payload interfaces.ExecutionData
+		var overrideBuilder bool
+		payload, fullBlobsBundle, overrideBuilder, err = vs.ExecutionEngineCaller.GetPayload(ctx, pid, slot)
 		switch {
 		case err == nil:
 			warnIfFeeRecipientDiffers(payload, feeRecipient)
-			return payload, blobsBundle, overrideBuilder, nil
+			return payload, overrideBuilder, nil
```

Comment:
do we not need to use the fullBlobsBundle variable here? does that variable need to be set?